iambic/core/models.py

```diff
@@ -695,9 +700,13 @@ class ExpiryModel(IambicPydanticBaseModel):
             "Upon being set to true, the resource will be deleted the next time iambic is ran."
         ),
     )
-    expires_at_default: Optional[Union[str, datetime.datetime, datetime.date]] = Field(
+    expires_at_default: Optional[str] = Field(
```

Comment:
it's actually dangerous to remove date time.datetime, and date time.date because a model definition means programatic contraction accepts it being a a valid type. 

When you remove daytime.datetime, and datetime.date, it means any API usage of expires_at_default using will not be valid anymore. 

I will suggest restoring the original stanza.

Reply:
Thank you for your feedback, @smoy. I will update my local repository to address the feedback.